### ISSUE TITLE
Applicability of Create Commons license

### ISSUE THREAD

**Issue by sander** [cadmiumkitty/togaf-content-metamodel-ontology#15]
Thanks @cadmiumkitty for this work! I’d like to reuse it, possibly extend under Creative Commons, and have a question about the following statement:

> Original from https://sites.google.com/site/ontologyprojects/home/togaf-core-content-metamodel is used with permission from Aurona Gerber. Subsequent modifications are released with Creative Commons Attribution-ShareAlike 4.0 International license.

Does this mean the current full ontology v2 is available for reuse under CC, or only the diff between the original work and the current version in this repo?

Not asking for legal advice, just to clarify about the scope of the permission from the original author.

**Comment by cadmiumkitty** [comment]
Thanks, @sander, I'm keen to see more people using it to build EA knowledge graphs!

Paraphrasing my email exchange with Aurona Gerber, I asked if she would be OK with releasing the original ontology under a friendly license such as Creative Commons Attribution or even Public Domain so I can pick it up and release improvements. I cited the licensing recommendation by OBO Foundry as a guide (http://www.obofoundry.org/principles/fp-001-open.html).

She said that I am welcome to build on the original ontology and improve it, so I proceeded with releasing it under Creative Commons Attribution (https://creativecommons.org/licenses/by/4.0/) and applying my updates on top.

In my mind, all of the content is covered by Creative Commons Attribution, and not just my updates. I'll update the README to clarify.

As you said, this is not a piece of legal advice, but I hope it helps.
